components/forecast/AvalancheTab.tsx

```diff
@@ -53,88 +52,66 @@ export const AvalancheTab: React.FunctionComponent<AvalancheTabProps> = React.me
+      style: {
+        fontSize: 16,
+        color: textColor,
+      },
+    },
+  };
+
   return (
-    <VStack space="4" bgColor={'#f0f2f5'}>
-      <Card paddingTop={2} borderRadius={0} borderColor="white" header={<Heading>Avalanche Forecast</Heading>}>
+    <VStack space="2" bgColor={'#f0f2f5'}>
+      <Card marginTop={2} borderRadius={0} borderColor="white" header={<Heading>Avalanche Forecast</Heading>}>
         <HStack justifyContent="space-evenly" space="2">
           <VStack space="2" style={{flex: 1}}>
             <Text bold style={{textTransform: 'uppercase'}}>
               Issued
             </Text>
-            <Text>{dateToString(forecast.published_time)}</Text>
+            <Text color="lightText">{dateToString(forecast.published_time)}</Text>
           </VStack>
           <VStack space="2" style={{flex: 1}}>
             <Text bold style={{textTransform: 'uppercase'}}>
               Expires
             </Text>
-            <Text>{dateToString(forecast.expires_time)}</Text>
+            <Text color="lightText">{dateToString(forecast.expires_time)}</Text>
           </VStack>
           <VStack space="2" style={{flex: 1}}>
             <Text bold style={{textTransform: 'uppercase'}}>
               Author
             </Text>
-            <Text>{forecast.author || 'Unknown'}</Text>
+            <Text color="lightText">{forecast.author || 'Unknown'}</Text>
           </VStack>
         </HStack>
       </Card>
-      <View style={styles.bound}>
-        <AvalancheDangerIcon style={styles.icon} level={highestDangerToday} />
-        <View style={styles.content}>
-          <Text style={styles.title}>THE BOTTOM LINE</Text>
-          <RenderHTML source={{html: forecast.bottom_line}} contentWidth={windowWidth} />
-        </View>
-      </View>
-      <AvalancheDangerTable date={parseISO(forecast.published_time)} current={currentDanger} outlook={outlookDanger} elevation_band_names={elevationBandNames} />
-      <Text style={styles.heading}>Avalanche Problems</Text>
-      {forecast.forecast_avalanche_problems.map((problem, index) => (
-        <AvalancheProblemCard key={`avalanche-problem-${index}`} problem={problem} names={elevationBandNames} />
-      ))}
-      <Text style={styles.heading}>Forecast Discussion</Text>
-      <View style={styles.discussion}>
-        <RenderHTML source={{html: forecast.hazard_discussion}} contentWidth={windowWidth} />
-      </View>
-      <Text style={styles.heading}>Media</Text>
+      <Card
+        borderRadius={0}
+        borderColor="white"
+        header={
+          <HStack space={2} alignItems="center">
+            <AvalancheDangerIcon style={{height: 32}} level={highestDangerToday} />
+            <Heading>The Bottom Line</Heading>
+          </HStack>
+        }>
+        <RenderHTML source={{html: forecast.bottom_line}} {...renderHTMLProps} />
+      </Card>
+      <Card borderRadius={0} borderColor="white" header={<Heading>Avalanche Danger</Heading>}>
```

Comment:
This `<Heading>Avalanche Danger</Heading>` is duplicated inside of the `<AvalancheDangerTable/>`

Reply:
AvalancheDangerTable needs major surgery

Reply:
Tracked in https://github.com/stevekuznetsov/avalanche-forecast/issues/45